helpdesk_mgmt_timesheet_addline/views/helpdesk_ticket_addline_view.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="hr_timesheet_line_form_addline" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.timesheet.view.form (addline)</field>
+        <field name="model">account.analytic.line</field>
+        <field name="arch" type="xml"> 
+            <form string="New Timesheet">
+                <sheet>
+                    <group>
+                        <field name="project_id" invisible="1"/>
```

Comment:
This is required due to 

https://github.com/odoo/odoo/blob/d8dc66a901bcf00e73f1c8048a71daf5e0bd321f/addons/analytic/models/analytic_account.py#L175-L179

and

https://github.com/odoo/odoo/blob/d8dc66a901bcf00e73f1c8048a71daf5e0bd321f/addons/project/models/project.py#L1030-L1032

in order to prevent error during installation

```
    odoo.tools.convert.ParseError: while parsing /home/david/Solvos/Desarrollo/CodeProjects/odoo-15-oca/slv-helpdesk/helpdesk_mgmt_timesheet_addline/views/helpdesk_ticket_addline_view.xml:3
    Error while validating view near:

    <form string="New Timesheet" __validate__="1">
                    <sheet>
                        <group>

    Field 'company_id' used in domain of field 'task_id' ([('company_id', '=', company_id), ('project_id.allow_timesheets', '=', True), ('project_id', '=?', project_id)]) must be present in view but is missing.

```


```suggestion
                        <field name="company_id" invisible="1"/>
                        <field name="project_id" invisible="1"/>
```